javacord-api/src/main/java/org/javacord/api/DiscordApi.java

```diff
@@ -87,6 +88,25 @@
      */
     AccountType getAccountType();
 
+    /**
+     * Gets the latest gateway latency.
+     *
+     * <p>To calculate the gateway latency, Javacord measures the time it takes for Discord to answer the gateway
+     * heartbeat packet with a heartbeat ack packet. Please notice, that this values does only get updated on every
+     * heartbeat and not on every method call. To calculate an average, you have to collect the latency over a period of
+     * multiple minutes.
+     *
+     * <p>In very rare cases, the latency will be {@code -1ns} directly after startup, because the initial heartbeat was
```

Comment:
Definitely a nit, but you should put a space between number and unit.
It reads better and follows the SI standard. :-)

```suggestion
     * <p>In very rare cases, the latency will be {@code -1 ns} directly after startup, because the initial heartbeat was
```